common/modifiers/wc_religion_modifiers.txt

```diff
@@ -0,0 +1,32 @@
+﻿recent_scarlet_mark_modifier = {
+	icon = piety_negative
+	same_faith_opinion = -20
+	zealot_opinion = -20
```

Comment:
should be strong, this is the scarlet crusade

Reply:
They already have the Scarlet Mark opinion. Combined this would be a opinion of -90. If you factor in the zealot opinion on the trait it would be -160. Make this an opinion modifier that decreases over time instead.